io.openems.edge.battery.fenecon.home/src/io/openems/edge/battery/fenecon/home/FeneconHomeBatteryImpl.java

```diff
@@ -234,25 +255,32 @@ protected ModbusProtocol defineModbusProtocol() throws OpenemsException {
 										ElementToChannelConverter.SCALE_FACTOR_MINUS_1) // [%]
 								.m(Battery.ChannelId.SOH, ElementToChannelConverter.SCALE_FACTOR_MINUS_1) // [%]
 								.build(), //
-						m(FeneconHomeBattery.ChannelId.CELL_VOLTAGE_MIN, new UnsignedWordElement(510)),
+						m(new UnsignedWordElement(510))//
+								.m(FeneconHomeBattery.ChannelId.CELL_VOLTAGE_MIN,
+										ElementToChannelConverter.DIRECT_1_TO_1)//
+								.m(Battery.ChannelId.MIN_CELL_VOLTAGE, ElementToChannelConverter.DIRECT_1_TO_1).build(), // [A]
```

Comment:
There is no need to keep the same information with the same precision in two channels

Reply:
I agree, this has remained the first prototype of the implementation.